### ISSUE TITLE
A11y_ApplicationInsights_WCAG_ApplicationInsightsConfiguration_Zoom: When zoomed to 400% contents on the screen gets distorted and not visible.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/vscode#104663]
"[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
Environment Details:​
Application Name: Application Insights
Operating System: Windows 10
Visual Studio: Version 16.6.0 Preview 2.0
​
Pre- Requisite:
On Windows, set your display resolution to 1280x1024, then use DPI zoom to 400%.(Windows Settings > Display Settings > Scale and Layout)

Repro Steps:​
1. Launch the Visual studio 2019 int preview 2.
2.Navigate to the project and select a web application and create a project.
3.Navigate to the solution explorer and select the web application and right click on it.
4.Observe if the end user is able to navigate to the desired screen after applying the zoom to 400%.

Actual:​
When users zoom the screen content to 400% contents on the screen gets distorted and not able to view the content in 400% zoom mode.

Expected:​
At 400% zoom user should be able to navigate to the  all the contents on the screen properly and  all the contents on the screen which are visible when the screen is at 100% zoom should be available even when it is zoomed to 400%.

User Impact:​
If the users zoom the content to 400%  and if controls are not visible on the screen then the user might miss out information that is available for the users with a normal 100% zoom screen.

Recommendations:​
Please refer to below URL which is repository for accessibility bug fixes.​
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx​

​MAS Reference:​
MAS1.4.10-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&cid=41453438-3aaf-4c66-8d6e-9b3cde934839

![104663_A11y_ApplicationInsights_WCAG_ApplicationInsightsConfiguration_Zoom](https://user-images.githubusercontent.com/64009219/90264795-3f1fb480-de6f-11ea-8b9b-0037b8cddc40.png)

**Comment by isidorn** [comment]
This is a bug tracker for VS Code, not Visual Studio int preview 2. Thus closing
